README.md

```diff
@@ -14,29 +14,33 @@ This library provides functions to test that a DAML application and its bots are
 
 #### Prerequisites
 
-To be able to develop the library you need:
-* sbt
-
-To use the library in a project:
-* download the library from |PUT LOCATION HERE| as a dependency
+To use the library in a project download the library from Maven Central as a dependency. In a Maven project, the following parts need to be present in pom.xml:
+In the `<dependencies> ... </dependencies>` section, one needs to add the library itself.
```

Comment:
This one is not needed, everyone using maven will know

Reply:
If you mean the "In the `<dependencies> ... </dependencies>` section, one needs to add the library itself." sentence, I would rather keep it, if not a problem